Issue143 (#146)
gravwell/wiki: ingesters/preprocessors/preprocessors.md

[Preprocessor "dropit"]
	Type=Drop               
```

## Cisco ISE Preprocessor

> The Cisco ISE logging system can be extremely difficult to work, this preprocessor is designed to help overcome some of the difficult challenges posed by both the format and transport of Cisco ISE logs.  See the [Cisco Introduction to ISE Syslogs](https://www.cisco.com/c/en/us/td/docs/security/ise/syslog/Cisco_ISE_Syslogs/m_IntrotoSyslogs.pdf) for more information.

Reviewer: ```suggestion
The Cisco ISE preprocessor is designed to parse and accomodate the format and transport of Cisco ISE logs.  See the [Cisco Introduction to ISE Syslogs](https://www.cisco.com/c/en/us/td/docs/security/ise/syslog/Cisco_ISE_Syslogs/m_IntrotoSyslogs.pdf) for more information.
```

> The Cisco ISE preprocessor is named `cisco_ise` and supports the ability to reassemble multipart messages, reformat the messages into a more usable format, filter unwanted message pairs, and remove redundant message headers.

Reviewer: ```suggestion
The Cisco ISE preprocessor is named `cisco_ise` and supports the ability to reassemble multipart messages, reformat the messages into a format more appropriate for Gravwell and modern syslog systems, filter unwanted message pairs, and remove redundant message headers.
```

### Attribute Filtering and Formatting

> The Cisco ISE logging system is designed to split a single message across multiple syslog messages.  The multipart message format for remote Cisco ISE syslog messages appears to be designed to support legacy systems that cannot appropriately handle larger syslog messages (even though the syslog specification can support very large messages).  Gravwell will handily accept messages that far exceed the maximum message size of syslog, however if you are supporting multiple targets for Cisco ISE messages is may be necessary to enable multipart messages.  Disabling multipart messages in your cisco device and letting Gravwell handle large payloads will be far more efficient.

Reviewer: ```suggestion
The Cisco ISE logging system is designed to split a single message across multiple syslog messages.  Gravwell will accept messages that far exceed the maximum message size of syslog, however if you are supporting multiple targets for Cisco ISE messages it may be necessary to enable multipart messages.  Disabling multipart messages in your cisco device and letting Gravwell handle large payloads will be far more efficient.
```
